storage.rules

```diff
@@ -0,0 +1,8 @@
+rules_version = '2';
```

Comment:
Why do we need this file? We have storage rules in the firebase settings, or do we need this file for the emulator?